package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gweninterpreter/gwen-web",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
Can you drop this change? I'll create a separate changelog+version bump commit after merging this

Reply:
Sure. done